chromium_src/ui/color/sys_color_mixer.cc

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "ui/color/sys_color_mixer.h"
+
+#include "brave/ui/color/brave_sys_color_mixer.h"
+
+#define kGrayscale kGrayscale) {                        \
+    AddGrayscaleSysColorOverrides(mixer, key);          \
+    ui::AddBraveGrayscaleSysColorOverrides(mixer, key); \
+  }                                                     \
+  else if (false
```

Comment:
I think it's better to override it via `#define AddSysColorMixer AddSysColorMixer_ChromiumImpl` so the override can be a bit simpler. Is it possible?

```
void AddSysColorMixer(ColorProvider* provider, const ColorProviderKey& key) {
  AddSysColorMixer_ChromiumImpl(provider, key);
  if (key.user_color_source == ColorProviderKey::UserColorSource::kGrayscale) {
    AddBraveGrayscaleSysColorOverrides(mixer, key);
  }
}
```

Reply:
I'd actually assumed this wouldn't work (as we'd be putting the overrides in a separate color mixer). However, it seems to turn out fine. 

Changed.